lib/elixir/lib/stream.ex

```diff
@@ -423,6 +423,10 @@ defmodule Stream do
   This operation will block the caller by the given interval
   every time a new item is streamed.
 
+  Do not use this function to generate number sequences. If
+  blocking the caller process is not necessary, instead use
+  `Stream.iterate(0, & &1 + 1)`.
```

Comment:
Probably we should have parens here, I think better to be closer to what `Macro.to_string/1` produce.